xmlschema/validators/schemas.py

```diff
@@ -321,11 +323,19 @@ def __init__(self, source: Union[SchemaSourceType, List[SchemaSourceType]],
                  build: bool = True,
                  use_meta: bool = True,
                  use_fallback: bool = True,
-                 loglevel: Optional[Union[str, int]] = None) -> None:
+                 loglevel: Optional[Union[str, int]] = None, use_translation=False,
+                 translation_folder: PurePath = None) -> None:
```

Comment:
If you need to keep the translation setting at each schema init you can collapse to a single extra argument, eg.: 

```python
use_translation: Union[None, Path, str] = None)
```